app/factory/Layer.js

```diff
@@ -246,6 +246,34 @@ Ext.define('CpsiMapview.factory.Layer', {
             }
             allFilters.push(bboxFilter);
 
+            /**
+             * Apply filters possibly set on columns of GeoExt FeatureGrid.
+             * To get access here these filters must be set on `filterchange`
+             * listener of FeatureGrid component like:
```

Comment:
This is just an example, isn't it? Filters may created by any component as well?

Reply:
Yes, I extended the comment with some additional infos